WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -943,11 +946,28 @@ public void onSiteChanged(OnSiteChanged event) {
         if (site != null) {
             mSelectedSite = site;
         }
+        if (getMySiteFragment() != null) {
+            getMySiteFragment().onSiteChanged(site);
+        }
     }
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
     public void onSiteRemoved(OnSiteRemoved event) {
         handleSiteRemoved();
     }
+
+    @Override public void onPositiveClicked(@NonNull String instanceTag) {
```

Comment:
The `@Override` annotation for the `onPositiveClicked` and `onNegativeClicked` are inline rather than separated here as well.  Sorry I missed these with my first review.